### PR TITLE
Android - Resize bottombar after window size changes (to always fill …

### PR DESCRIPTION
**Pull request by Cristo86**
…the screen horizontally)

Solves problem in devices with navigation bar (no physical home/back buttons) where an screen resize happens when starting up and may make the bottom bar not fill the screen and position itself incorrectly.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8084/app_12665_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8084/HighFidelity-Beta-PR12665-79ff2df860defa5715335a10d9fc1f5c5e334a1c.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8084/HighFidelity-Beta-PR12665-79ff2df860defa5715335a10d9fc1f5c5e334a1c.exe

File: interface/resources/qml/hifi/+android/bottombar.qml [hunk 1]
```diff
@@ -104,13 +104,26 @@ Item {
...
+    }
+
+    function onWindowGeometryChanged(rect) {
+        relocateAndResize(rect.width, rect.height);
+    }
 
     Component.onCompleted: {
         // put on bottom
-        width = Window.innerWidth;
         height = 255;
```
Comment (SamGondelman) [inline interface/resources/qml/hifi/+android/bottombar.qml]: Could you put this up at the top of the Item definition (line 28) so it's clearer where relocateAndResize gets its height from?

Comment (Cristo86) [reply interface/resources/qml/hifi/+android/bottombar.qml]: I'd move this line:

`height = 255;`

As it's a plain property and value right?

Comment (SamGondelman) [reply interface/resources/qml/hifi/+android/bottombar.qml]: Yeah, I think you should be able to do `height: 255` on line 28.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8093/app_12665_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8093/HighFidelity-Beta-PR12665-e6a8b322b113483218bb84fd556b4c4b02d56837.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8093/HighFidelity-Beta-PR12665-e6a8b322b113483218bb84fd556b4c4b02d56837.exe

**Pull request merged**